app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentSheet.kt

```diff
@@ -41,6 +41,7 @@ class RemoveAppointmentSheet : BottomSheetActivity() {
   @Inject
```

Comment:
Reword commit to `Add "Patient already visited" to remove appointment options`?

Reply:
Reworded